app/views/reasonableExcuseJourneys/other/noJs/UploadFirstDocumentPage.scala.html

```diff
@@ -65,7 +66,7 @@ <h1 class="govuk-heading-l">@messages("otherReason.uploadEvidence.heading")</h1>
             @fileUploadDialog(form)
         </div>
         <p>
-            @govukButton(Button(content = Text(messages("otherReason.uploadEvidence.noJs.button.uploadFile"))))
+            <a href="@nextPageWithUpload" class="govuk-button">@messages("otherReason.uploadEvidence.noJs.button.uploadFile")</a>
```

Comment:
<a> tags do not submit the form. They are hyperlinks that redirects the user to a page without submitting the form itself. You shouldn't need to actually pass anything here and can be removed.